marklogic-data-hub-central/ui/src/components/entities/entity-tiles.tsx

```diff
@@ -178,10 +196,35 @@ const EntityTiles = (props) => {
           }
     }
 
-    const outputCards = (entityType, mappingCardData, matchingCardData) => {
-        let output;
+    const getCustomArtifacts = async () => {
+        try {
+            if(props.canReadCustom){
+                let response = await axios.get('/api/steps/custom');
+                if (response.status === 200) {
+                    let entArt = response.data;
+                    entArt.sort((a, b) => (a.entityType > b.entityType) ? 1 : -1)
```

Comment:
This seems like something the backend should do. I feel a single DS endpoint should do all the work here - return the object structure I mentioned above, and do any sorting necessary.